cmake/environmentchecks.cmake

```diff
@@ -25,9 +25,11 @@ if (EMSCRIPTEN)
 else ()
   find_package(Python ${PREFERRED_PYTHON_VERSION} COMPONENTS Interpreter Development)
 
-  find_program(BUILDCACHE_EXE buildcache)
+  if(NOT WIN32 OR MSVC)
```

Comment:
not clear what we are trying to exclude here.

Reply:
Agreed that it's not needed as such, hence removed in commit 627b5162.